cdap-watchdog/src/test/java/co/cask/cdap/logging/KafkaTestBase.java

```diff
@@ -94,4 +117,64 @@ private static Properties generateKafkaConfig() throws IOException {
 
     return prop;
   }
+
+  private static void waitForKafkaStartup(String zkConnectString, long timeoutMills) throws Exception {
```

Comment:
There is a easier way to wait for Kafka service to start. You can use `ZKBrokerService` to help:

```java
final BrokerService brokerService = new ZKBrokerService(zkClient);
brokerService.startAndWait();
Tasks.waitFor(true, new Callable<Boolean>() {
  public Boolean call() throws Exception {
    return Iterables.size(brokerService.getBrokers());
  }
}, 10, TimeUnit.SECONDS, 100, TimeUnit.MILLISECONDS);
```